media-libs/netpbm/netpbm-10.88.01.recipe

```diff
@@ -606,11 +626,16 @@ REQUIRES="
 	lib:libpng$secondaryArchSuffix
 	lib:libtiff$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	cmd:xml2_config
+	lib:libx11$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
 	netpbm${secondaryArchSuffix}_devel = $portVersion compat >= 10
-	devel:libnetpbm$secondaryArchSuffix = 11.68 compat >= 11
```

Comment:
libversion is 11.88 now

Reply:
Check the lib version in the package (or in the source after build), it's **libnetpbm.so.11.88**